libdlib/package.xml

```diff
@@ -7,7 +7,7 @@
 	</description>
 	<author email="florian.jordan@ipa.fraunhofer.de">Florian Jordan</author>
 	<maintainer email="richard.bormann@ipa.fraunhofer.de">Richard Bormann</maintainer>
-	<license>LGPL</license>
+	<license>Boost Software License</license>
```

Comment:
I was told to use the same licence as the library, because it provides the source code of it